rust/lance/src/dataset.rs

```diff
@@ -590,9 +599,9 @@ impl Dataset {
         self.append_impl(batches, params).await
     }
 
-    /// Get the base URI of the dataset.
-    pub fn uri(&self) -> &Path {
-        &self.base
+    /// Get the fully qualified URI of this dataset.
+    pub fn uri(&self) -> &str {
+        &self.uri
     }
```

Comment:
This is a breaking change to a public API. I've labelled it as such.

However, if we didn't want to make a breaking change, you could instead make a new method and deprecate this one.

Reply:
Let's bump the version. The rust API does not have a large audience yet
https://crates.io/crates/lance/reverse_dependencies

This new release process does not allow us to accumulate a few breaking changes together for a new version? 
https://github.com/lancedb/lance/blob/main/release_process.md#breaking-change-policy this is suggesting i manually bump the version, right?

Reply:
> This new release process does not allow us to accumulate a few breaking changes together for a new version?

You can accumulate them. Once you've bumped the version once, you don't have to do it again until the next release.